rsts/deployment/sandbox.rst

```diff
@@ -186,32 +102,43 @@ We'll proceed like with :ref:`locally hosted flyte <deploy-sandbox-local>` with
 Flyte configuration on your remote cluster.
 
 
-#. The Flyte sandbox can be deployed via a helm chart. From the root dir of the flyte repo run ::
+#. Add Helm repo for flyte ::
+
+    helm repo add flyteorg https://helm.flyte.org
+
+#. Install Flyte dependency helm chart (this will install the minio, Postgres, Kubernetes-dashboard, and contour) ::
 
-    helm repo add flyte https://flyteorg.github.io/flyte
-    helm install -n flyte -f values.yaml --create-namespace flyte flyte/flyte
+    helm install -n flyte flyte-deps flyteorg/flyte-deps --create-namespace -f https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-sandbox.yaml
 
-#. For customizations instructions, see ``/charts/flyte/README.md`` in the flyte repo.
+#. Install flyte-core chart ::
 
-#. You can now port-forward (or if you have load-balancer enabled then get an LB) to connect to remote FlyteConsole, as follows ::
+    helm install flyte flyteorg/flyte-core -n flyte -f https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-sandbox.yaml --wait
 
-    kubectl port-forward --address 0.0.0.0 svc/flyte-contour-envoy 30081:80 -n flyte
+#. Verify Flyte deployment using the following command ::
 
-#. Open the console http://localhost:30081/console.
+    kubect get pods -n flyte
+
+.. note::
+
+    Make sure all pods are in Running condition. If you see anything that's crashing, check them in this order: postgres, minio, flyteadmin, datacatalog, flytepropeller.
+
+#. Get the URL of the ingress service ::
```

Comment:
Looks like because of formatting of the note block above, this shows as `1.` instead of `5.` I think you need to indent the note block.